0016/endpoint-time-series-event.avsc

```diff
@@ -0,0 +1,102 @@
+{
+    "namespace":"org.kaaproject.ipc.collector2ts.gen.v1",
+    "name":"TimeSeriesEvent",
+    "type":"record",
+    "fields":[
+        {
+            "name":"correlationId",
+            "type":"string",
+            "doc":"Message ID primarily used to track message processing across services"
+        },
+        {
+            "name":"timestamp",
+            "type":"long",
+            "doc":"Message creation UNIX timestamp in milliseconds"
+        },
+        {
+            "name":"timeout",
+            "type":"long",
+            "default":0,
+            "doc":"Amount of milliseconds since the timestamp until the message expires. Value of 0 is reserved to indicate no expiration."
+        },
+        {
+            "name":"appVersionName",
+            "type":"string",
+            "doc":"Application version name the data is sent for"
+        },
+        {
+            "name":"endpointId",
+            "type":"string",
+            "doc":"Identifier of the endpoint that data points belong to"
+        },
+        {
+            "name":"timeSeriesName",
+            "type":"string",
+            "doc":"Name of the time series that data points belong to"
+        },
+        {
+            "name":"dataPoints",
```

Comment:
We use "data samples" in both 2/DCP and 13/DRP.

Reply:
Data samples are different from data points:
- **Endpoint data sample**: block of endpoint-related data of arbitrary structure.
- **Data point**: data structure consisting of a timestamp, a value, and optional key-value tags.
Data point value and tags value can be of arbitrary primitive or composite data type.

Reply:
oh, ok. still, a bit confusing

Reply:
I can add to the data point definition that it is not to be confused with data sample :)

Reply:
Maybe call it "time series sample"?

Reply:
@Slash32 @ityshko what do you think?